packages/store/src/__tests__/adapter/markdown.unit.spec.ts

```diff
@@ -0,0 +1,338 @@
+import type { Root } from 'mdast';
+import { expect, test } from 'vitest';
+
+import { MarkdownAdapter } from '../../adapter/index';
+import type { BlockSnapshot } from '../../transformer/type';
+
+test('test markdown adapter', async () => {
```

Comment:
Using this article here looks a bit verbose and it may be updated afterwards. So having a short markdown test content with basic heading, list, inline style and quote inside should be adequate.